app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title do "Forgot Password" end %>
+
+<% content_for :alerts do render "application/alerts" end %>
```

Comment:
For a one-liner, you can do `content_for :alerts do render "application/alerts"`.